docs/PIPELINE.md

```
@@ -0,0 +1,64 @@
+backlog weeks):
+
+```bash
+python scripts/pipeline-status.py
+python scripts/pipeline-status.py --weekly   # markdown for daily-mine
+```
+
+Terms match [`docs/terminology.md`](terminology.md).
+
+## Flow
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)